datacube/utils/xarray_geoextensions.py

```diff
@@ -7,23 +7,25 @@
 with Data Cube by Monkey Patching those classes.
 
 This extension is reliant on an `xarray` object having a `.crs` property of type
-:class:`datacube.utils.geometry.CRS`. This is used to inspect the spatial dimensions of the
+:class:`odc.geo.crs.CRS`. This is used to inspect the spatial dimensions of the
 :class:`Dataset` or :class:`DataArray`, and provide new attributes for accessing a
-:class:`datacube.utils.geometry.GeoBox`, affine transform and extent for the dataset as
+:class:`odc.geo.geobox.GeoBox`, affine transform and extent for the dataset as
 `.geobox`, `.affine` and `.extent` respectively.
 
 """
 import warnings
 import xarray
-from datacube.utils import geometry, spatial_dims
+from datacube.utils import spatial_dims
 from datacube.utils.math import affine_from_axis
+from odc.geo import CRS, CRSError, wh_
+from odc.geo.geobox import GeoBox
 
```

Comment:
One more thing that I have missed in the previous review. This file should just delegate geobox extraction to `.odc.geo.xr` with a warning, this extraction logic is in `odc-geo`.

Reply:
I've implemented the change; let me know if it aligns with what you had in mind.